apps/condo/domains/user/components/auth/InputPhoneForm.tsx

```diff
@@ -22,6 +21,8 @@ import { START_CONFIRM_PHONE_MUTATION } from '@condo/domains/user/gql'
 
 
 import { RegisterContext } from './RegisterContextProvider'
+import { LoginWithSBBOLButton } from '../../../common/components/LoginWithSBBOLButton'
+import { isSafeUrl } from '../../../common/utils/url.utils'
```

Comment:
You should not use relative paths that have many hops to the parent directory.
Use alias `@condo/...`